src/SearchBox.jsx

```diff
@@ -76,18 +78,47 @@ var HitList = React.createClass({
 
 var SearchBox = React.createClass({
 
+    componentDidMount: function () {
+        document.addEventListener('mousedown', this.handleClickOutside);
```

Comment:
This does not seem to work when clicking map?

Reply:
No, apparently this event is not firing on the map, probably because leaflet has its own listeners. Maybe its best to leave the code like this, and add the option to manually call "closeHits()" on the component, so that the users of the component can listen specifically to  map clicks themselves, and then call the "closeHits()" function? Either that, or I think the only way is to make a div that covers the whole screen, and listen to clicks on that div instead. 

What do you think is best?

Reply:
Or I could also add button that closes the menu.

Reply:
The option of exposing closeHits seems like a good idea.